safespace/src/App.scss

```diff
@@ -1,47 +1,250 @@
 
+$dark-green: #173A3E;
+$sand: #E0BD6B;
+$grey : #e6e6e6;
+$iphone-grey: #828489;
 
 .App {
   text-align: center;
+  // background-image: url('http://clipart-library.com/data_images/348839.png'); doesn't render why?
+}
+
+button {
+  background-color: black;
+  border: 1px solid black;
+  color: $sand;
+  padding: 10px;
+  border-radius: 10px;
+  margin: 5px;
+    &:hover {
+      cursor: pointer
+    }
 }
 
 .safespace {
-  border: 2rem solid pink;
-  padding: 50px;
   min-height: 100vh;
-  margin: 2rem;
-}
+  background-color: $sand;
 
-.message {
-  border: 5px solid pink;
-  margin: 10px;
-}
+  .mood {
+    display: flex;
+    flex-direction: column;
 
-.scroll {
-  width: 100px;
-}
+    h2 {
+      margin: 20px;
+      color: $dark-green;
+      font-size: 3rem;
+    }
+
+    button {
+      background-color: $dark-green;
+      font-size: 2.5rem;
```

Comment:
on elements that will change color or background colors of positions on hover or active or focus states, you may want to add a transition property to the element outside the psuedo state, adding a easing time of the change so it isn't so jarring to the end user, makes for a nicer UX (user experience). here is the basic transition property i add to things that change on hover...  transition: all .3s ease-in-out      this would be added to the main elements styles, not the hover styles...so it would go in the styles of button before you do the &:hover styles